bin/birdhouse

```diff
@@ -98,6 +99,9 @@ print_config_command() {
     configs_cmd_suffix="${configs_cmd_suffix} unset BIRDHOUSE_LOCAL_ENV ;"
   fi
 
+  if [ "${option}" = "no-suffix" ]; then
+    configs_cmd_suffix=
+  fi
```

Comment:
If `$(print_config_command)` was used, wouldn't `if [ -z "$1" ]; then` work the same? 

Just curious. `no-suffix` is fine, but seems like a roundabout way with unnecessary hardcoded values.

Reply:
By using a constant like 'no-suffix', it allows us to easily add other options in the future as well. For this specific use case `[ -z "$1" ]` would work as well but it's not as friendly to future developers. 

I also like `no-suffix` because it's self-documenting :shrug: